eos-updater-flatpak-installer/eos-updater-flatpak-installer-fallback.timer.in

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Endless OS Fallback Flatpak Installer Timer
+Documentation=man:eos-updater-flatpak-installer(8)
+ConditionKernelCommandLine=!endless.live_boot
+ConditionKernelCommandLine=!eos-updater-disable
+
+[Timer]
+# After 5 minutes the network connection has likely been set up, but we don't
+# have a hard requirement
```

Comment:
`/etc/NetworkManager/dispatcher.d/50-eos-phone-home` may be an inspiration for triggering this on network connection, if desired

Reply:
Not sure if we do want to block on there being a network connection. It’s feasible that eufi could fix some problems just by re-running in the new OSTree (so with a new libostree, etc.)